lib/src/components/NxBinaryDonutChart/NxBinaryDonutChart.tsx

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, { forwardRef } from 'react';
+import { Props, propTypes } from './types';
+export { Props } from './types';
+
+import './NxBinaryDonutChart.scss';
+
+const NxBinaryDonutChart = forwardRef<SVGSVGElement, Props>(
+    function NxBinaryDonutChart(props, ref) {
+      const innerRadiusPercent = props.innerRadiusPercent === 0 ? props.innerRadiusPercent :
+        (props.innerRadiusPercent || 50);
+
+      const strokeWidth = 100 - innerRadiusPercent;
+      const r = innerRadiusPercent + (strokeWidth / 2);
+      const strokeDasharray = [(100 - props.percent), props.percent];
+      return (
+        <svg ref={ref} viewBox="-100 -100 200 200" {...props}>
+          <circle className="nx-binary-donut-chart-circle"
+                  strokeWidth={strokeWidth}
+                  r={r}>
+          </circle>
+          <circle className="nx-binary-donut-chart-circle"
```

Comment:
So this actually represents the amount of the the donut that's not filled in, right?

Reply:
> So this actually represents the amount of the the donut that's not filled in, right?

Yeah your arc class suggestion makes it clearer now I think https://github.com/sonatype/sonatype-react-shared-components/pull/172/commits/0b2fad02e32cf9927810950d59c52abe24ac9a2f